src/Data/Nat/DivMod.agda

```diff
@@ -222,11 +203,13 @@ m/n≤m m n = *-cancelʳ-≤ (m / n) m n (begin
   m           ≤⟨ m≤m*n m n ⟩
   m * n       ∎)
 
-m/n<m : ∀ m n .{{_ : NonZero m}} .{{_ : NonZero n}} → n ≥ 2 → m / n < m
-m/n<m m n n≥2 = *-cancelʳ-< _ (m / n) m (begin-strict
-  (m / n) * n ≤⟨ m/n*n≤m m n ⟩
-  m           <⟨ m<m*n m n n≥2 ⟩
-  m * n       ∎)
+m/n<m : ∀ m n .{{_ : NonZero m}} .{{_ : NonTrivial n}} →
+        let instance _ = Nat.nonTrivial⇒nonZero n in m / n < m
```

Comment:
So unfortunately this is a breaking change to the lemma, so we can't make it v2.1. Will have to wait for v3.0 I'm afraid.

Reply:
Ah! Happy to revert this change, and return to it later, if that means the rest can go through?